### ISSUE TITLE
several differences on some parameters

### ISSUE THREAD

**Issue by payphone131** [YapengTian/AV-Robustness-CVPR21#5]
hello, i am wondering:
why the frequency when extracting waveforms is 11025hz in readme.md but in extract_audio.py it is 11000hz?
why the recommended epochs of AVE dataset is 100 in your paper but in train_defense_AVE.sh it is 30?
why the learning rate of frame and audio of training defense on AVE are 1e-3 and 1e-4 respectively in your paper but in train_defense_AVE.sh they are 1e-4 and 1e-3?

**Comment by YapengTian** [comment]
(1) In the dataloader, the audio is resampled at 11025hz. I commonly used the audio extraction script in my projects and did not specify it as 11025. Feel free to modify it. I guess it would not affect much.
(2) I found the model can reproduce competitive results with 30-epoch training. For saving trianing time, I set it as 30. 
(3) The order should be wrong in the appendix. As shown in the code, the frame learning rate is 1e-4 and sound learning rate is 1e-3 since we used a pre-trained visual net and the audio net was learned from scratch. 

Thanks for your questions! Please feel free to ask any other questions.

**Comment by payphone131** [comment]
thanks a lot for your reply! got another question to ask if you don't mind. i am not sure about what i should do when training defense on AVE. in your train_defense_AVE.sh there are weights_Dv, weights_Da, weights_sound, weights_frame, weights_classifier, does it mean that i should get these files by executing other training procedure before executing train_defense_AVE.sh?

**Comment by YapengTian** [comment]
Just run  ./scripts/train_attack_AVE.sh to obtain these (the so-called attack training in the code is actually training an audio-visual event recognition model; the attack only occurred during attack evaluation)

**Comment by payphone131** [comment]
thanks! your reply is very clear.
